packages/amplify-app/src/index.js

```diff
@@ -338,6 +339,35 @@ async function createAndroidHelperFiles() {
   }
 }
 
+async function createIosHelperFiles() {
+  const configFile = path.join(process.cwd(), '/amplifyxc.config');
+  const configStr = 'push=false\nmodelgen=false\nprofile=default\nenvName=amplify';
+  const awsConfigFile = path.join(process.cwd(), '/awsconfiguration.json');
+  const amplifyConfigFile = path.join(process.cwd(), '/amplifyconfiguration.json');
+  const amplifyDir = path.join(process.cwd(), '/amplify');
+  const configJsonData = '{}';
+  const configJsonObj = JSON.parse(configJsonData);
+  const configJsonStr = JSON.stringify(configJsonObj);
+
+  // Write files if needed and them to xcode project if one exists
+  if (!fs.existsSync(configFile)) {
+    fs.writeFileSync(configFile, configStr);
+  }
+  await addFileToXcodeProj(configFile);
+  if (!fs.existsSync(awsConfigFile)) {
```

Comment:
may be add a empty line before this to make it easy to grok the code

Reply:
added